quickwit-indexing/src/actors/packager.rs

```diff
@@ -205,7 +208,7 @@ fn create_packaged_split(
         .sum();
 
     // Extracts tag values from `_tags` special fields.
-    let mut tags = BTreeSet::default();
+    let mut field_name_to_tag_values: HashMap<String, HashSet<String>> = HashMap::default();
```

Comment:
ok we should not do that :)

you should use the inverted index to get the number of tokens.